docs/cookbooks/harness.rst

```diff
@@ -631,6 +649,7 @@ information such as:
 2. Provide connection pool information per device
 3. Overwrite devices alias
 4. Which connection type to prioritize for the run
+5. Support for custom connections
```

Comment:
```suggestion
5. Support for multiple connections per context(cli/yang/etc) with new schema
```

Reply:
Do we need the 'with new schema' ?

Reply:
updated @tahigash. @dwapstra Kindly let me know if there is a change needed.